bundles/org.openhab.core.semantics/model/SemanticTags.csv

```diff
@@ -100,8 +100,10 @@ Equipment,KitchenHood,Fan,Kitchen Hood,Kitchen Hoods,
 Equipment,HVAC,,HVAC,"Heating, Ventilation, Air Conditioning, A/C, A/Cs, AC",
 Equipment,Inverter,PowerSupply,Inverter,Inverters,
 Equipment,LawnMower,Horticulture,Lawn Mower,Lawn Mowers,
-Equipment,LightSource,,Light Source,"Lightbulb, Lightbulbs, Bulb, Bulbs, Lamp, Lamps, Lights, Lighting",
-Equipment,LightStrip,LightSource,Light Strip,"LightStripe, LED Strip",
+Equipment,LightSource,,Light Source,"Lights, Lighting",
+Equipment,Lightbulb,LightSource,Light Bulb,"Lightbulbs, Bulb, Bulbs, Lamp, Lamps",
+Equipment,LightStripe,LightSource,Light Strip,Light Stripes,
```

Comment:
I don't know where the labels are used, but having the same label could potentially be confusing. For example a UI that presents a list of tag labels, you'll see two entries of `Light Strip` and you won't know which one to choose.